packages/matrix-client-server/src/matrixDb/matrixDbTestConf.json

```diff
@@ -0,0 +1,20 @@
+{
+  "base_url": "http://example.com/",
+  "cron_service": false,
+  "database_engine": "sqlite",
+  "database_host": "./src/__testData__/test.db",
+  "database_vacuum_delay": 7200,
+  "is_federated_identity_service": false,
+  "key_delay": 3600,
+  "keys_depth": 5,
+  "mail_link_delay": 7200,
+  "rate_limiting_window": 10000,
+  "server_name": "matrix.org",
+  "smtp_sender": "yadd@debian.org",
+  "smtp_server": "localhost",
+  "template_dir": "./templates",
+  "userdb_engine": "sqlite",
+  "userdb_host": "./src/__testData__/test.db",
+  "matrix_database_engine": "sqlite",
+  "matrix_database_host": "./src/__testData__/testMatrix.db"
+}
```

Comment:
Such file must be in a __something__ directory _(test only)_

Reply:
Moved it to the __testData__ folder !